src/rebound.h

```diff
@@ -297,6 +297,11 @@ struct reb_integrator_trace {
     struct reb_vec3d com_vel;
 
     int* current_Ks; // Tracking K_ij for the entire timestep
+    int* temp_Ks;    // temporary K array for adding/removing particles 
```

Comment:
This one still needs to be removed (by using an array temporarily allocated then freed in particle.c until we can wrap our head around how to modify the array in place)